gossip/exception_policy.py

```diff
@@ -8,7 +9,11 @@ class ExceptionPolicy(object):
     @contextmanager
     def context(self):
         ctx = TriggerContext()
-        yield ctx
+        try:
+             yield ctx
+        except:
```

Comment:
Should be `except Exception:` I think... we don't want KeyboardInterrupts and SystemExits to mess with it...